plugins/labhub.py

```diff
@@ -381,3 +381,88 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'^migrate\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/+issues/(\d+)\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)/*$',  # Ignore LineLengthBear, PyCodeStyleBear
+               # Ignore LineLengthBear, PyCodeStyleBear
+               re_cmd_name_help='migrate <complete-source-issue-URL> <complete-target-repo-URL>',
+               flags=re.IGNORECASE)
+    def migrate_issue(self, msg, match):
+        """
+        Migrate an issue from source repo
+        to target repo owned by the org
+        """
+        orig_host = match.group(1)
+        org = match.group(2)
+        repo_name_orig = match.group(3)
+        issue_number = match.group(4)
+        user = msg.frm.nick
```

Comment:
it is a bit strange seeing this variable mixed in amoung all the regex matches.

please read every line of your diff and ask yourself if it can be improved.